hadoop-hdds/docs/content/design/tools.md

```diff
@@ -0,0 +1,136 @@
+    - `ozone debug om prefix`
+        The new location of `ozone debug prefix`.
+- `ozone debug datanode`
+    A subcommand for all debug operations that run locally on a datanode instance.
+    - `ozone debug datanode container`
+        The new location of `ozone debug container` and all of its subcommands. These commands are used to inspect container replicas locally on datanodes. Currently `container` is the only debug subcommand on the datanodes.
+- `ozone debug replicas`
+    Currently all online debug tools fit under this category, where they get a listing from the OM and perform checks on the replica information retrieved. The help message for this subcommand should explicitly state that it runs over the network against a running cluster.
+    - `ozone debug replicas chunk-info`
+        This is the new location of `ozone debug chunkinfo`
+    - `ozone debug replicas verify [--checksums] [--padding] [--blocks]`
+        This is the combination of `ozone debug find-missing-padding`, `ozone debug read-replicas`, and the [proposed block metadata check tool](https://github.com/apache/ozone/pull/7548). The command takes  paths in the namespace and performs checks on the objects based on the flags given:
+      - No flags: Error, at least one check must be specified.
+      - `--blocks` checks block existence of all replicas as proposed in [HDDS-11891](https://issues.apache.org/jira/browse/HDDS-11891).
+      - `--padding` Checks padding for EC keys only. No-op for Ratis keys.
+      - `--checksums` Pulls data and verifies the checksums on the client side for all replicas. This is a thorough check but it will run very slowly.
+- `ozone debug ratis`
+    This subcomand is where we can put Ratis related commands that are specific to Ozone. For bringing the Ratis shell into Ozone, see the [`ozone ratis`](#ratis) subcommand.
+    -  `ozone debug ratis parse <--role={om,scm,datanode}>`
+        Currently our only Ozone-specific subcommand, this can be used to parse any Ratis on-disk state and apply the Ozone specific schema to it if needed. The command should be able to figure out whether it is looking at a log segment, raft-meta, etc on it's own, but the user will need to specify what type of Ozone role the Ratis information came from so it can apply the proper schema.
+
+### Repair
+
+Invoked as `ozone repair`, these commands make modifications to an Ozone instance. For now, there is no plan to support repair operations online or over the network, but this could be implemented in the future. All commands will support a `--dry-run` flag to show what they will do without making any modifications. Subcommand grouping follows similar guidelines to `ozone debug`.
+
+- `ozone repair om`
+    These commands make repairs to an individual OM instance.
+    - `ozone repair om quota`
+        The new location of `ozone repair ldb quota`
```

Comment:
`ozone repair ldb quota` does not exist currently, but I found this:

```
Usage: ozone repair quota [COMMAND]
Operational tool to repair quota in OM DB.
Commands:
  status  CLI to get the status of last trigger quota repair if available.
  start   CLI to trigger quota repair.
```

which is an online command (requires running OM).